include/aspect/utilities.h

```diff
@@ -135,7 +135,12 @@ namespace aspect
      * @param [in] allow_missing_keys Whether to allow that some keys are
      *   not set to any values, i.e. they do not appear at all in the
      *   @p key_value_map. This also allows a completely empty map.
-     *
+     * @param [in] use_values_instead_of_differences Whether to define the
+     *   phase properties based on the exact values rather than value differences.
+     *   if this is false, then all the phase properties need to be entered by
+     *   a value for the first phase followed by M-1 values for the differences
+     *   between adjacent phases. Also note that the quotients rather than the
```

Comment:
Please explain here what you mean by quotients exactly.

Reply:
Changed to the "differences" in "log values"